frontend/src/modules/fluenteditor/components/RichTranslationForm.js

```diff
@@ -466,14 +482,19 @@ export default class RichTranslationForm extends React.Component<EditorProps> {
             return null;
         }
 
+        let attributeLabel = null;
         if (!label) {
             label = 'Value';
```

Comment:
I think the code could be refactored a bit to make this assignment later on, in `renderElements`.